tests/testthat/test-filters.R

```diff
@@ -54,3 +54,100 @@ test_that("get_survey_filters gets available filter options and sorts them", {
   expect_equal(get_survey_filters(NULL), list())
   expect_equal(get_age_filters(data.frame(survey_id = NULL)), list())
 })
+
+test_that("can construct sorted tree from data frame", {
+  data <- data.frame(
+    id = c("MWI", "MWI.1", "MWI.2", "MWI.1.1", "MWI.1.2"),
+    parent_id = c(NA, "MWI", "MWI", "MWI.1", "MWI.1"),
+    sort_order = c(1, 2, 3, 4, 5),
+    stringsAsFactors = FALSE
```

Comment:
you could use `data_frame` and reduce noise here

Reply:
Nice - have fixed that throughout.